firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -13,3 +13,23 @@ bool alignPiece(int col, int row)
     return true;
 }
 
+// Assumes that the starting position is the center of the square the piece is on
+void centerPiece()
+{
+    // Loop counter
+    int i;
+
+    moveToFirstCircle();
+
+    // Enable electromagnet
+    analogWrite(ELECTROMAGNET, PWM_SCALE);
+
+    for (i = 0; i < NUM_CIRCLES; i++)
+    {
+        makeCircle(i);
+        moveToNextCircle(i);
```

Comment:
`moveToNextCircle(i % 4)`

Reply:
The function does need to know the circle number.